pkg/actuators/machineset/controller.go

```diff
@@ -111,7 +112,10 @@ func reconcile(machineSet *machinev1.MachineSet) (ctrl.Result, error) {
 	}
 	instanceType, ok := InstanceTypes[providerConfig.InstanceType]
 	if !ok {
-		return ctrl.Result{}, mapierrors.InvalidMachineConfiguration("unknown instance type: %s", providerConfig.InstanceType)
+		klog.Error("Unable to set scale from zero annotations: unknown instance type: %s", providerConfig.InstanceType)
+		klog.Error("Autoscaling from zero will not work. To fix this, manually populate machine annotations for you instance type\n- %s\n- %s\n -%s",
+			cpuKey, memoryKey, gpuKey)
+		return ctrl.Result{}, nil
```

Comment:
Worth adding a comment here to explain why we don't return an error here, so we don't accidentally put it back later